radix-transactions/tests/test_manifest_compiler_error_diagnostics.rs

```diff
@@ -1,15 +1,19 @@
 use radix_common::network::NetworkDefinition;
-use radix_transactions::manifest::blob_provider::*;
-use radix_transactions::manifest::compiler::*;
+use radix_transactions::manifest::*;
 
 macro_rules! check_manifest {
-    ($manifest:expr, $blob_provider:expr, $style:expr) => {{
+    ($manifest_kind:expr, $manifest:expr, $blob_provider:expr $(,)?) => {{
         let manifest = include_str!(concat!("assets/", $manifest, ".rtm"));
         let diagnostic = include_str!(concat!("assets/", $manifest, ".diag"));
 
-        let err = compile(manifest, &NetworkDefinition::simulator(), $blob_provider).unwrap_err();
-
-        let x = compile_error_diagnostics(manifest, err, $style);
+        let x = compile_any_manifest_with_pretty_error(
+            manifest,
+            $manifest_kind,
+            &NetworkDefinition::simulator(),
+            $blob_provider,
+            CompileErrorDiagnosticsStyle::PlainText,
```

Comment:
I very much like the idea of switching to plain text, diagnostic files are much more readable.
But maybe it would make sense to leave 1 or 2 tests with TextTerminalColors just to makes sure it works as expected, when we eg. update `annotate-snippets` crate.

Reply:
I'd support adding a separate explicit test for this, but that said:
* I'd hope `annotate-snippets` test `Renderer::styled()` themselves, so I don't feel strongly that we need to test it
* I doubt we'll be updating `annotate-snippets` regularly